packages/components/src/Components/ErrorState/DefaultErrorMessage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const DefaultErrorMessage = ({redirectLink}) => {
+    const redirectLink = 'https://access.redhat.com/support';
```

Comment:
you're redeclaring this var. Just use a default prop instead on the parent component and pass it here

Reply:
oh I must've missed that on the prop, it was meant to just be declared on the DefaultMessage component since the parent doesn't need it. Just took it off. 

Would the parent having that as a default prop be preferable?